CMakeLists.txt

```diff
@@ -34,10 +34,11 @@
 # constrained by the version of CMake available on target systems.
 cmake_minimum_required(VERSION 3.25 FATAL_ERROR)
 
+set(CMAKE_PROJECT_VERSION 0.0.1)
```

Comment:
Exciting! Do we want to start with `0.0.1` or `0.1.0`?

Reply:
I think starting with `0.0.1` is fine until we remove the beta tag